nh_eobs_mobile/models/nh_clinical_form_description.py

```diff
@@ -12,16 +12,25 @@ def to_dict(cls, model):
         field_utils = cls.pool['nh.clinical.field_utils']
         obs_fields = field_utils.get_obs_fields_from_model(model)
         for field in obs_fields:
-            field_description = {
-                'name': field.name,
-                'type': field.type,
-                'label': field.string,
-                'selection': field.selection,
-                'selection_type': 'text',
-                'initially_hidden': False,
-                'required': str(field.required).lower(),
-                'necessary': str(field.necessary).lower()
-            }
+            if field.type != 'datetime':
```

Comment:
Not convinced this is required. The changes below in nh_observations/models/nh_clinical_field_utils.py should allow the field type to be identified without the need for this.